e2e/setup-e2e.sh

```
@@ -318,11 +310,16 @@ setup_test_environment() {
         # The operator uses nodeport mode by default with port 8082
         local BASEDOMAIN=$(kubectl get jumpstarter -n "${JS_NAMESPACE}" jumpstarter -o jsonpath='{.spec.baseDomain}')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)